Integrations/integration-Malwr.yml

```diff
@@ -0,0 +1,346 @@
+        username = demisto.params()['username']
+        password = demisto.params()['password']
+    else:
+        username = None
+        password = None
+
+    malwr = MalwrAPI(
+        url=demisto.params()['server'],
+        username=username,
+        password=password
+    )
+
+    entry = {
+        'Type': entryTypes['note'],
+        'ContentsFormat' : formats['text'],
+        'ReadableContentsFormat' : formats['text']
+    }
+
+    if __name__ == '__main__':
+        if demisto.command() == 'test-module':
+            demisto.results('ok')
+            sys.exit(0)
+
+        elif demisto.command() == 'malwr-submit':
+            file_id = demisto.args()['fileId']
+            filepath = get_file_path(file_id)
+            res, soup = malwr.submit_sample(filepath)
+            if 'analysis_link' in res:
+                analysis_id = res['analysis_link'].split('/')[-2]
+
+                message = 'File submitted: %s%s\n' % (MAIN_URL, res['analysis_link'])
+                message += 'MD5: %s\n' % (res['md5'], )
+                message += 'Analysis ID: %s' % (analysis_id, )
+
+                entry['Contents'] = str(soup)
+                entry['HumanReadable'] = message
+                entry['EntryContext'] = {'Malwr.Submissions(val.Id==obj.Id)': {'Id': analysis_id, 'Md5': res['md5'], 'Status': 'pending'}}
+
+            else:
+                entry['HumanReadable'] = res
+
+        elif demisto.command() == 'malwr-status':
+            analysis_id = demisto.args()['analysisId']
+            status, data, soup = malwr.get_status(analysis_id)
+            if status == 'complete':
+                message = 'The analysis is complete, you can view it at: %s%s.' % (MAIN_URL, data)
+            elif status == 'pending':
+                message = 'The analysis is still in progress.'
+            else:
+                message = 'Error: the specified analysis does not exist.'
+
+            entry['Contents'] = str(soup)
+            entry['HumanReadable'] = message
+            entry['EntryContext'] = {'Malwr.Submissions(val.Id==obj.Id)': {'Id': analysis_id, 'Status': status}}
+
+        elif demisto.command() == 'malwr-result':
+            analysis_id = demisto.args()['analysisId']
+            status, is_malicious, soup, md5 = malwr.get_result(analysis_id)
+
+            if status == 'pending':
+                message = 'The analysis is still in progress.'
+                demisto.results(message)
+                sys.exit(0)
+
+            if is_malicious:
+                entry['EntryContext'] = {'Malwr.Submissions(val.Id==obj.Id)': {'Id': analysis_id, 'Status': status, 'Malicious': {'Vendor': 'Malwr'}}}
+                entry['EntryContext']['DBotScore'] = {'Indicator': md5, 'Vendor': 'Malwr', 'Score': 3 if is_malicious else 0}
+                message = 'The file is malicious.'
+            else:
+                entry['EntryContext'] = {'Malwr.Submissions(val.Id==obj.Id)': {'Id': analysis_id, 'Status': status}}
```

Comment:
It only adds the Malicious field under the Malwr entry, where the id matches the given one. So it's not an entire new entry.From what I know this is the way it's usually done